astroplan/tests/test_core.py

```diff
@@ -155,3 +155,34 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+def test_pang():
```

Comment:
I'd prefer `test_parallactic_angle` over `test_pang`.
Easier to read and clearer 1:1 correspondence with the function being tested by name.